### PR TITLE
Symfony compat fixes

### PR DESCRIPTION
**Pull request by fabpot**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,17 +1,29 @@
...
+env:
+    global:
+        - deps=no
+
+before_install:
+    - composer self-update
 
 install:
-    - composer install
+    - if [ "$deps" = "no" ]; then composer --prefer-source --dev install; fi;
```
Comment (xabbuh) [inline .travis.yml]: `--dev` is the default behaviour.

Comment (fabpot) [inline .travis.yml]: I know but it was like that before.

**Comment by kriswallsmith** [comment]
Thanks @fabpot!

**Comment by fabpot** [comment]
@kriswallsmith @stof Can you make a release with this fix?

**Comment by stof** [comment]
sure

**Comment by stof** [comment]
2.6.1 has been released

**Pull request merged**